langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -222,6 +274,11 @@ public type JsonFloat ()|boolean|string|float|JsonFloat[]|map<JsonFloat>;
 #
 # Returns an error if the string cannot be parsed.
 #
+# ```ballerina
+# string jsonFloatStr = "12.12";
+# value:JsonFloat jsonFloat = check jsonFloatStr.fromJsonFloatString();
```

Comment:
This again, is more useful for members of a structure.

E.g.,
```ballerina
string jsonFloatStr = "[12, true, 123.4, \"hello\"]";
value:JsonFloat jsonFloat = check jsonFloatStr.fromJsonFloatString();
```

Basically, all the numeric values will be parsed as floats. I think we can write a meaningful example with a mapping.